spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -91,8 +92,14 @@
 				<version>1.0.4</version>
 			</dependency>
 
-			<!--Starters-->
+      <!-- spring-cloud-gcp-starter-config -->
+      <dependency>
```

Comment:
Nit: checkstyle isn't complaining probably because this is an xml file. Can you fix the indentation here?